middlewares/account.middleware.js

```diff
@@ -166,15 +166,19 @@ async function inviteAccount(req, res, next) {
             message: Constants.Error.EMAIL_500_MESSAGE,
         });
     }
-
+}
+async function getInvites(req, res, next) {
```

Comment:
Function comments - also what does find do when the query is empty?

Reply:
Query is not an input parameter, so it is always set to {}.